src/main/java/com/keeper/homepage/domain/calendar/application/ScheduleService.kt

```diff
@@ -0,0 +1,36 @@
+package com.keeper.homepage.domain.calendar.application
+
+import com.keeper.homepage.domain.Schedule.entity.Schedule
+import com.keeper.homepage.domain.calendar.dao.ScheduleRepository
+import com.keeper.homepage.domain.calendar.dao.ScheduleTypeRepository
+import lombok.RequiredArgsConstructor
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+class ScheduleService(
+    val scheduleRepository: ScheduleRepository,
+    val scheduleTypeRepository: ScheduleTypeRepository,
+) {
+
+    @Transactional
+    fun saveSchedule(name: String, startTime: LocalDateTime,
+        endTime: LocalDateTime, scheduleTypeId: Long,
+    ): Long {
```

Comment:
```suggestion
    fun saveSchedule(
        name: String, 
        startTime: LocalDateTime,
        endTime: LocalDateTime, 
        scheduleTypeId: Long,
    ): Long {
```
너무나 사소하지만... 저는 이렇게 포맷팅하지 않으면 불편한 병에 걸렸습니다 ㅎㅎ 🤣 (코틀린 권장 스타일이긴 하더라구요!)